cloud/IoTHubListener/EventHubListener.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.EventHubs;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Devices;
+using Google.Protobuf;
+using HeartbeatProto;
+
+namespace GWManagementFunctions
+{
+    public static class EventHubListener
+    {
+        [FunctionName("EventHubListener")]
+        public static async Task Run(
+                [EventHubTrigger(
+                    "messages/events",
+                    Connection = "EventHubIngestConnectionString")]
+                EventData[] events,
+                [EventHub("dest", Connection="EventHubEndpointConnectionString")]
+                IAsyncCollector<string> tsiEventHub,
+                DateTime enqueuedTimeUtc,
+                ILogger logger)
+        {
+            var exceptions = new List<Exception>();
+
+            foreach (EventData message in events)
+            {
+                try
+                {
+                    // capture current processing time 
+                    var azFncInitializedTime = DateTime.UtcNow;
+
+                    // Get expiration setting in minutes
+                    int MessageExpirationTimeinMinutes;
+                    if(!Int32.TryParse(
+                                Environment.GetEnvironmentVariable(
+                                    "MessageExpirationTimeinMinutes"),
+                                out MessageExpirationTimeinMinutes)) {
+                        MessageExpirationTimeinMinutes = 5;
+                    }
+
+                    // initialize IoT Hub Service CLient, this uses an adapter to wrap the 
+                    // service client code for testability
+                    var iotHubServiceClient = ServiceClient.CreateFromConnectionString(
+                            Environment.GetEnvironmentVariable("IoTHubConnectionString"),

```

Comment:
Feels like we need to call out the dependency on the IoT Hub connection string to do the "close the loop" pong to the edge more explicit. Got any ideas?